jpo-ode-svcs/src/main/java/us/dot/its/jpo/ode/coder/AbstractCoder.java

```diff
@@ -86,18 +88,21 @@ public void decodeFromStreamAndPublish(InputStream is, String topic) throws IOEx
    }
 
    @Override
-   public void publish(String topic, String msg) {
+   public void publishJson(String msg) {
```

Comment:
No need to rename the overloaded methods. Besides, publishJson is misleading. This method has no idea the msg is JSON. It just knows that it is a String and it publishes it as such. If yo want to rename it, call it publishString.